app/config.py

```diff
@@ -58,8 +58,8 @@ class Config(object):
         config = LdConfig(
             sdk_key = LD_CLIENT_KEY,
             base_uri = os.environ.get("LD_RELAY_URL"),
-            events_uri = os.environ.get("LD_RELAY_URL"),
-            stream_uri = os.environ.get("LD_RELAY_URL")
+            events_uri = os.environ.get("LD_RELAY_EVENTS_URL", base_uri),
```

Comment:
@levlaz based on this:
```
            base_uri = os.environ.get("LD_RELAY_URL"),
            events_uri = os.environ.get("LD_RELAY_URL"),
            stream_uri = os.environ.get("LD_RELAY_URL")
```

and events_uri being unset, I think it would point to `events.launchdarkly.com` and not the relay as it does now if that is set. So the behavior in the case of my change is:
`events_uri -> base_uri -> events.launchdarkly.com`

If we get rid of that, it will be:
`events_uri -> events.launchdarkly.com`

I'm good with either setup, just let me know if I should still update the PR to get rid of the `base_uri` default value for events and stream

Reply:
I am really confused. :( 

Let's pair on this later on.

Reply:
@InTheCloudDan I am confusing myself. This is perfect, thanks for knocking this out.